ledger/src/blockstore_options.rs

```diff
@@ -36,6 +44,8 @@ impl BlockstoreOptions {
             recovery_mode: None,
             enforce_ulimit_nofile: false,
             column_options: LedgerColumnOptions::default(),
+            num_rocksdb_compaction_threads: default_num_compaction_threads(),
+            num_rocksdb_flush_threads: default_num_flush_threads(),
```

Comment:
could we use `..BlockstoreOptions::default()` here?

Reply:
Yeah probably, I was just pattern matching the pre-existing stuff but we can use `..BlockstoreOptions::default()`; the only thing unique here is setting `enforce_ulimit_nofile = false`